mobile/src/components/shared/ICardPage.jsx

```diff
@@ -153,7 +162,9 @@ const styles = StyleSheet.create({
   notification: {
     flexDirection: 'row',
     borderRadius: 30,
-    width: 312,
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    //width: '100%',
```

Comment:
We should probably remove these commented styles unless we need them somewhere else

Reply:
Someone else had formatted the ICardPage but it wasn't properly formatted on my screen(idk if it looks better on someone else's screen), especially with the new messages and stuff that has been added with this pr. So I kept them in case those are the better options for the future

Reply:
I think we should delete them, worst case we can check the repo history.